ark/utils/data_utils.py

```diff
@@ -23,17 +22,13 @@ def generate_deepcell_input(data_xr, data_dir, nuc_channels, mem_channels):
             membrane channels to be summed over
     """
     for fov in data_xr.fovs.values:
-        out = np.zeros((data_xr.shape[1], data_xr.shape[2], 2), dtype=data_xr.dtype)
+        out = np.zeros((2, data_xr.shape[1], data_xr.shape[2]), dtype=data_xr.dtype)
```

Comment:
Let's add a check to make sure that either nuc_channels or mem_channels is specified, and raise an error if both are None

Reply:
added